web/src/utils/atlas/fetchUser.ts

```diff
@@ -0,0 +1,50 @@
+import { gql, type GraphQLClient } from "graphql-request";
+import { toast } from "react-toastify";
+
+import { OPTIONS } from "utils/wrapWithToast";
+
+type GetUserResponse = {
+  user: {
+    email: string;
+  };
+};
+
+export type User = {
+  email: string;
+};
+
+const query = gql`
+  query GetUser($address: String!) {
+    user(address: $address) {
```

Comment:
address is no longer an input value. By default this query will search for the address that was used to login.